src/inject/inject.js

```diff
@@ -9,10 +9,10 @@ chrome.extension.sendMessage({}, function() {
       clearInterval(readyStateCheckInterval);
       chrome.runtime.onMessage.addListener(routeCommand);
     }
-  }, 10);
+  }, 3);
 
   function routeCommand(request) {
-    card = $('.active-card');
+    card = $('.card-detail-window');
```

Comment:
So this actually breaks one of my favourite pieces of functionality.

I would mouseover a card in the board and press "copy short url". It means I didn't have to wait for cards to load to get their URL for someone else.

Maybe a fallback would be enough? `$('.card-detail-window') || $('.active-card')`